gatorgrade/output/output_percentage_printing.py

```diff
@@ -0,0 +1,25 @@
+"""Output with the percentage of checks that the student has met requirments."""
+from colorama import Fore
+
+
+def print_percentage(results):
+    """Print percentage acts as fuction that will produce the output."""
+    # iterate through results tuples
+    true_list = []  # empty list for storing true results
+    for i in results:
```

Comment:
It would probably be best to introduce descriptive variable names in your for loops as well instead of `i`, maybe `result` could be used to stay consistent.

Reply:
I'm going to leave the variable as i and write some comments describing the logic.